chat/message/src/MessageLinks/MessageLinks.styles.ts

```diff
@@ -0,0 +1,23 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+
+export const baseStyles = css`
+  container-type: inline-size;
```

Comment:
Do you know what browsers we want to support and if this is compatible with those browsers?

Reply:
Container queries should work on all (updated) modern browsers + we will fall back to the two column layout when it's unsupported

https://caniuse.com/css-container-queries